reposilite-plugins/javadoc-plugin/src/main/kotlin/com/reposilite/plugin/javadoc/JavadocResponse.kt

```diff
@@ -0,0 +1,13 @@
+package com.reposilite.plugin.javadoc
+
+
+class JavadocResponse(contentType: String, response: Any?) {
```

Comment:
Just

```kotlin
data class JavadocResponse(val contentType: String, val response: Any?)
```

Reply:
Also, what's the `response`, can we slightly precise this type or we really want to support everything here?